tsql/TSqlParser.g4

```diff
@@ -2489,6 +2490,24 @@ backup_service_master_key
          ENCRYPTION BY PASSWORD EQUAL encryption_password=STRING
     ;
 
+kill_statement
+    : KILL (kill_process | kill_query_notification | kill_stats_job)
+    ;
+
+// https://docs.microsoft.com/en-us/sql/t-sql/language-elements/kill-transact-sql
+kill_process
+    : (session_id=(DECIMAL|STRING) | UOW) (WITH STATUSONLY)?
+    ;
+
+// https://docs.microsoft.com/en-us/sql/t-sql/language-elements/kill-query-notification-subscription-transact-sql
+kill_query_notification
+    : QUERY NOTIFICATION SUBSCRIPTION (ALL | subscription_id=DECIMAL)
+    ;
+
+// https://docs.microsoft.com/en-us/sql/t-sql/language-elements/kill-stats-job-transact-sql
+kill_stats_job
+    : STATS JOB job_id=DECIMAL
```

Comment:
Do you consider the element label `job_id` is necessary here? `DECIMAL` is easily reachable in Visitor. The same not for the rules above.

Reply:
Not functionally no, I added it to make the generated grammar more descriptive/discoverable without needing to reference the source grammar file.

```csharp
var context = new TSqlParser.Kill_stats_jobContext();
var jobId1 = context.DECIMAL();
var jobId2 = context.job_id;
```

Is it a good practice to not generate an extra field if it's not functionally required? _I'm still reading the book..._

Reply:
Actually not so important, but I prefer as such a minimal grammar as possible.

Reply:
I'll keep that in mind, thanks.